### PR TITLE
add deprecation warning

### PR DESCRIPTION
**Pull request by chamer81**
Added a message to use image-cleanup instead of this tool.

### CODE REVIEW

**Comment by srueg** [comment]
LGTM, will merge it as soon as https://github.com/appuio/container-oc/pull/37 is merged.

**Comment by anothertobi** [comment]
> LGTM, will merge it as soon as [appuio/container-oc#37](https://github.com/appuio/container-oc/pull/37) is merged.

Better merge it now so that it will be in the image built when https://github.com/appuio/container-oc/pull/37 is merged.

**Comment by srueg** [comment]
There is a hook in this repo which will trigger a rebuild of the `appuio/oc` image immediately...

**Comment by anothertobi** [comment]
Didn't know that 🙈

File: cleanup/cleanup.sh [hunk 1]
```diff
@@ -94,6 +94,7 @@ get_deletion_candidates() {
 }
 
 main() {
+    echo 'Deprecated: Please use appuio:image-cleanup.'
```
Comment (ccremer) [inline cleanup/cleanup.sh]: `Please use appuio:image-cleanup.` This is very confusing for me. Is it referencing a docker image on Docker hub, or a github project? I would change it to `Please use github.com/appuio/image-cleanup.`

File: cleanup/cleanup.sh [hunk 2]
```diff
@@ -94,6 +94,7 @@ get_deletion_candidates() {
 }
 
 main() {
+    echo 'Deprecated: Please have a look at github.com/appuio/image-cleanup.'
```
Comment (bittner) [inline cleanup/cleanup.sh]: I'd opt for "Use foo instead", but no hard feelings, really.

**Pull request merged**